scriptmodules/supplementary/autostart.sh

```diff
@@ -102,20 +105,29 @@ function remove_autostart() {
 
 function gui_autostart() {
     cmd=(dialog --backtitle "$__backtitle" --menu "Choose the desired boot behaviour." 22 76 16)
+    local has_pegasus=0
+    local has_kodi=0
+
+    command -v pegasus-fe >/dev/null && has_pegasus=1
+    command -v kodi-standalone >/dev/null && has_kodi=1
+
     while true; do
         if isPlatform "x11"; then
             local x11_autostart
             if [[ -f "$home/.config/autostart/retropie.desktop" ]]; then
-                options=(1 "Autostart Emulation Station after login (Enabled)")
+                options=(1 "Autostart EmulationStation after login (Enabled)")
                 x11_autostart=1
             else
-                options=(1 "Autostart Emulation Station after login (Disabled)")
+                options=(1 "Autostart EmulationStation after login (Disabled)")
                 x11_autostart=0
             fi
         else
             options=(
-                1 "Start Emulation Station at boot"
-                2 "Start Kodi at boot (exit for Emulation Station)"
+                1 "Start EmulationStation at boot"
+            )
+            [    $has_kodi == 1 ] && options+=(2 "Start Kodi at boot (exit starts EmulationStation)")
+            [ $has_pegasus == 1 ] && options+=(3 "Start Pegasus at boot")
```

Comment:
Please use double `[[` (I don't think the spacing is necessary - we don't do that elsewhere).

Reply:
If these variables are 0/1 I would also use `-eq`

Reply:
OK, used `-eq` since it's consistent with the rest of the module.